README.md

```diff
@@ -177,7 +177,7 @@ To protect the community, all contributors are required to sign the Apache Softw
 [Tips for Developing the Spark Cassandra Connector](doc/developers.md)
 
 Checklist for contributing changes to the project:
-* Create a [SPARKC JIRA](https://datastax-oss.atlassian.net/projects/SPARKC/issues)
+* Create a [SPARKC JIRA](https://issues.apache.org/jira/projects/CASSANALYTICS)
```

Comment:
Do we want to add another Jira component before we point folks at CASSANALYTICS? I suppose we can move things over later, so maybe not a big deal.

Reply:
pushed a fix